tests/php/Controller/SignalingControllerTest.php

```diff
@@ -591,10 +591,6 @@ public function testBackendRoomInvitedPublic() {
 			->willReturn($room);
 
 		$participant = $this->createMock(Participant::class);
-		$room->expects($this->once())
-			->method('getParticipantBySession')
-			->with($sessionId)
-			->willThrowException(new ParticipantNotFoundException());
 		$room->expects($this->once())
 			->method('getParticipantBySession')
```

Comment:
we tested the same function twice and that's why the tests failed before the last push